medical/models/medical_patient.py

```diff
@@ -95,6 +95,15 @@ def action_invalidate(self):
             ('z', 'law marriage'),
         ])
 
+    @api.multi
+    def onchange_state(self, state_id):
+        '''
+        Hack to allow for onchange_state on partner.
+        Uses current user's partner, as there should be no correlation
+        between the partner and the response from this method anyways
+        '''
+        return self.env.user.partner_id.onchange_state(state_id)
```

Comment:
Woot that works, thanks @pedrobaeza - really just needed to make sure there wasn't some sort of convenient method already provided by the API. Seems every time I break down in simple stuff, someone else has already added a method for it (also why I love Odoo).

Implemented, tested, pushed. As always, your advice is much appreciated :smile: